roles/lnls-ans-role-sirius-apps/defaults/main.yml

```diff
@@ -48,12 +48,50 @@ sirius_apps_dev_packages:
     repo_version: "{{ pkg_version_siriuspy }}"
     clone_path: /tmp
     install_chdir: siriuspy
-    install_via_setup: true
-    setup_extra_args: "--no-deps"
-    setup_state: forcereinstall
-    repo_cvs: git+
+    install_via_makefile: true
```

Comment:
Just a doubt. Why are we preferring `Makefile` instead of `setup`? Isn't it better to use setup?

Reply:
We will standardize to use Makefile targets. It seems more flexible and general. Did you take a look at one of the install targets?

Reply:
Pure makefiles (without a build system actually generating them, such as autotools, cmake, etc) are difficult to maintain I think, as they can contain virtually anything, from system commands, downloading packages, compiling things, etc. It would be difficult to keep track of everything that is being installed and done on a host.

On the other hand, system packages, such as debian packages (or pip packages or even "setup" packages) must follow a more strict set of rules, in which is easier to maintain. Also, ideally, we should be driving away from compiling things on a host system. It forces the host to have the build tools, takes longer to install packages and, in the end, it makes the installation more fragile.

Reply:
As a final comment. In my opinion we should standardize, or try to, using debian packages, or at least a packaging system that is widely known. Makefiles could be used for development (but even for that I think we should strive for a more standard build system, cmake, autotools, setuptools, etc), but we shouldn't, in my opinion, use them for distributing our code: it lacks features of a package manager.

Reply:
Points taken. We use makefile in order to force certain actions necessary when setuptools _install_ and _develop_ are used interchanbly, which is still necessary since the packages are still evolving. 

_trackcpp_ package needs compilation. At this point everything is already installed in hosts that allows this. But it would be nice to have a debian package for it, instead. I am afraid the implementation overhead for this at this point is too costly, no? But we should try it!

Reply:
I think once the environment for debian packetization is complete, maintaining it should be easier. But sure, setting everything up takes some time. We will keep that in mind and discuss more. I'm approving this, as we won't make any of these changes right now.